### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by githubjeka**
yii2-stubs-generator better move to `require-dev` instead `require` section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The preferred way to install this extension is through [composer](http://getcomp
 Either run
 
 ```
-php composer.phar require --prefer-dist bazilio/yii2-stubs-generator
+php composer.phar require --dev --prefer-dist bazilio/yii2-stubs-generator:"*"
```
Comment (samdark) [inline README.md]: Why `*`? There are releases.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ or add
 "bazilio/yii2-stubs-generator": "*"
```
Comment (samdark) [inline README.md]: Should not be `*`.

File: README.md (hunk 1 above)
Comment (githubjeka) [inline README.md]: `0.0.3` or `*@stable`?

Comment (samdark) [inline README.md]: Don't specify any version. Composer will decide for itself.

File: README.md (hunk 2 above)
Comment (samdark) [inline README.md]: `~0.0.3` is a good idea.

**Comment by bazilio91** [comment]
Thank you! :+1:

**Pull request merged**